content/zh-tw/pod.md

```diff
@@ -0,0 +1,33 @@
+---
+title: Pod
+status: Completed
+category: 概念
+tags: ["基礎設施", "基本原理", ""]
+---
+
+## 是什麼 {#what-it-is}
+
+Pod 是 Kubernetes 環境中最基本的可部署單位。
+它代表部署和管理容器化應用程式的必要組件。
+每個 Pod 包含一個應用程式實例，並且可以由一個或是多個[容器](/zh-tw/container/) 所組成。
+Kubernetes 將 Pod 視為大規模部署的一部分並且可根據需求[垂直](/zh-tw/vertical-scaling/)或是[水平](/zh-tw/horizontal-scaling/)地擴展 Pod 的規模
+
+## 解決的問題 {#problem-it-addresses}
+
+雖然容器通常作為獨立的單位運行以及控制特定的工作負載，
+但是某些情況下，容器間需要以緊密耦合的方式進行互動與被控制。
+
+如果將這些密切相關的容器單獨管理，則會導致冗余的管理任務出現。
+舉例來說，操作者必須反覆地確保這些容器彼此的部署狀態以確保他們保持在一起。
+此外這些密切相關的容器其生命週期必須一致同步的被處理，但是卻只能各別單獨管理。
+
+## 如何幫助 {#how-it-helps}
+
+Pods 將緊密關係的容器們打包成一個單一的單位，此舉大幅度的簡化了容器操作。
```

Comment:
```suggestion
Pod 將緊密關係的容器們打包成一個單一的單位，此舉大幅度的簡化了容器操作。
```
Traditional Chinese does not have plural nouns, it is better to change them to singular nouns

Reply:
Done